undertow/src/main/java/org/wildfly/extension/undertow/deployment/UndertowDeploymentProcessor.java

```diff
@@ -444,16 +442,6 @@ private void processDeployment(final WarMetaData warMetaData, final DeploymentUn
             }
         }
 
-
-        // OSGi web applications are activated in {@link WebContextActivationProcessor} according to bundle lifecycle changes
-        if (isWebappBundle) {
-            UndertowDeploymentService.ContextActivatorImpl activator = new UndertowDeploymentService.ContextActivatorImpl(builder.install());
-            deploymentUnit.putAttachment(ContextActivator.ATTACHMENT_KEY, activator);
-            deploymentUnit.addToAttachmentList(Attachments.BUNDLE_ACTIVE_DEPENDENCIES, deploymentServiceName);
```

Comment:
we could probably also drop Attachments.BUNDLE_ACTIVE_DEPENDENCIES constant as well as Attachments.OSGI_MANIFEST

Reply:
Yes, we will do it. The constants you mention above are not used in wildfly.
I have another PR for wildfly core ready just didn't sent it.
We need to get these changes to wildfly first otherwise we would break wildfly with it.